conanfile.py

```diff
@@ -0,0 +1,76 @@
+import os
+from conans import ConanFile, CMake, tools
+
+
+class CoseCConan(ConanFile):
+    name = "cose-c"
+    version = "20200225"
+    license = "BSD"
+    url = "https://github.com/cose-wg/COSE-C"
+    description = """;"""
+    topics = ("COSE")
+    settings = "os", "compiler", "build_type", "arch"
+    options = {
+        "shared": [True, False],
+        "use_embedtls": [True, False]
+    }
+    default_options = {
+        "shared": False,
+        "use_embedtls": False
+    }
+    
+    generators = "cmake", "cmake_find_package"
+
+    _cmake = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    @property
+    def _build_subfolder(self):
+        return "build_subfolder"
+
+    def source(self):
+        self.run(
+            "git clone -b build--improve-cmake git@github.com:gocarlos/COSE-C.git")
```

Comment:
Does this need changing now that I have merged in over there?

Reply:
I'll change this